### DISCUSSION TITLE
Meteoswiss opendata in 2026

### DISCUSSION THREAD

**Discussion by libertykite** [open-meteo/open-meteo#716] [General]
Hi @patrick-zippenfenig,

I've been dreaming of having access to meteo swiss weather data for a while and I don't know if you've seen but things are starting to happen on their side. There is an article that describes the project: https://www.meteoswiss.admin.ch/services-and-publications/service/open-government-data.html
And there is a github (https://github.com/MeteoSwiss/publication-opendata/tree/main#readme) which is used to ask users about their future needs. I wouldn't be able to say what I need, but I would dream that Swiss weather data was available on Openmeteo :-)

**Comment by patrick-zippenfenig** [comment]
Thanks for sharing. Yes, as a Swiss resident I would also be interested in weather model data from MeteoSwiss. Ideally the MeteoSwiss ICON models could be made available as open-data. I talked to others at MeteoSwiss and the open-data process will be rather slow... As soon as gridded data is available, I will add it to Open-Meteo

**Comment by fayo-del** [comment]
Hi the Swiss weather open data program has recently evolved and now exposes additional datasets and APIs. Would it be possible to add support for Swiss weather data in Open‑Meteo?  

edit: I just looked at the source and saw a "swissmeteo" section. Is Swiss data already implemented, and is there a way to follow the progress on this feature?

PS thanks for the work on Open‑Meteo.

**Reply by terraputix** [reply]
Hi,

ICON-CH1 and ICON-CH2 models are already integrated into Open-Meteo. Compare: https://open-meteo.com/en/docs/meteoswiss-api
